config/symbols.pspeu.txt

```diff
@@ -184,7 +184,7 @@ D_8003C708 = 0x091e1308; // size:0x8
 g_BgLayers = 0x091E1328; // size:0x300
 g_Tilemap = 0x091e1628; // size:0x50
 g_CurrentEntity = 0x091E1678;
-g_Entities = 0x091e1680;
+g_Entities = 0x091E1680; // size:0xBC
```

Comment:
I may be wrong, but I think I tried this before and splat considers this to be a duplicate range with the `PLAYER_*` symbols below? I don't think it really supports having both at the moment from memory.

Reply:
It didn't complain when I ran it locally and the tests are good, so I dunno.

Reply:
Splat will print a warning about the range, but it still works. It would be nice if it handled "overlay" symbols, though.